test/integration_test_helper.rb

```diff
@@ -194,9 +195,22 @@ def save_edition_and_assert_success
     end
   end
 
-  def save_edition_and_assert_success_without_ajax
+  def save_edition_and_assert_success_slow
     save_edition
-    assert_save_attempted(false)
+
+    save_attempted = (page.has_selector?('.workflow-message-saving', text: 'Saving') ||
+                      page.has_selector?("input[type=submit]#save-edition:enabled"))
+    assert save_attempted, "Failed to attempt saving the edition"
+
+    saved = (page.has_content?("edition was successfully updated.") ||
+             page.has_css?('.workflow-message', text: 'Saved'))
+    assert saved, "Failed to save the edition"
+  end
+
+  def save_edition_and_assert_success_without_ajax
+    with_javascript = false
```

Comment:
Wouldn't that make it just say `save_edition(false)` on the next line?  I think that would be quite hard to read.